.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
```

Comment:
Why is CI limited to only PRs? Given that it takes some time to run, it's great if CI is already "don" when a PR gets submitted so any fixes can be addressed quickly.

Also: Pushes to plugin repos happen rarely. I don't think there is a cost concern.

Reply:
Do you think should run on branch? If we send multiple commits during the day in practice we consume lots of miniutes.

Reply:
I would argue that it's worth the money. Once a PR is submitted, all new commits will trigger CI anyway. This only cuts the costs for branches that are still in development. Having direct feedback from CI is helpful.